generated/nidigital/nidigital/session.py

```diff
@@ -3200,7 +3767,7 @@ def read_sequencer_register(self, reg):
     def reset_device(self):
         r'''reset_device
 
-        TBD
+        Returns a digital pattern instrument to a known state. method
```

Comment:
It would be good to copy the details for `reset` and `reset_device` from LabVIEW documentation:
https://www.ni.com/documentation/en/ni-digital/17.5/ni-digital-node-ref/reset/
https://www.ni.com/documentation/en/ni-digital/17.5/ni-digital-node-ref/reset-device/

Reply:
Done